sympy/functions/elementary/hyperbolic.py

```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
     r"""
     The hyperbolic sine function, `\frac{e^x - e^{-x}}{2}`.
 
-    * sinh(x) -> Returns the hyperbolic sine of x
+    * sinh(x) -> Returns the hyperbolic sine of x.
+
+    Examples
+    ========
+
+    >>> from sympy import sinh
+    >>> sinh(1.2)
```

Comment:
Let's not use floating point numbers for examples. The examples should show that the functions are symbolic.

Reply:
Changed.